### DISCUSSION TITLE
Alternative Views

### DISCUSSION THREAD

**Discussion by bharrislo** [inkle/ink#699] [Ideas]
I like using Ink because it's user friendly and dynamic, but I often find myself getting confused as to where my choices lead and it's time consuming to click through everything on the right pane. Have y'all considered adding a flowchart view that visually maps your story?

**Comment by JasonWarrenUK** [comment]
Heya, I doubt they'll do that because @joningold has spoken often about how a flowchart soon becomes impossible to manage/unhelpful once you start using a lot of the capabilities of ink

I sometimes build charts if diagrams to help me visualise some stuff, but I found I actually didn't need to have a 1:1 correspondence with my entire code.
